snuba/utils/streams/batching.py

```diff
@@ -161,6 +161,15 @@ def signal_shutdown(self) -> None:
     def _handle_message(self, msg: Message[TPayload]) -> None:
         start = time.time()
 
+        self.__metrics.timing(
+            "receive_latency",
+            (start - msg.timestamp.timestamp()) * 1000,
+            tags={
+                "topic": msg.partition.topic.name,
+                "partition": str(msg.partition.index),
```

Comment:
I guess we do not need to add the consumer group as a tag because, being this before processing, it does not really matter ?

Reply:
It should already be set as a tag on the metrics constructor:

https://github.com/getsentry/snuba/blob/ed6d794dfc60fdc0adb21d523858f47d530350d9/snuba/consumers/consumer_builder.py#L96-L98

https://github.com/getsentry/snuba/blob/ed6d794dfc60fdc0adb21d523858f47d530350d9/snuba/cli/subscriptions.py#L185-L188

(This is another thing that will need to be consolidated.)